src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/entities/task/Task.java

```diff
@@ -256,5 +268,12 @@ public Boolean getWarnings() {
         return hasWarnings;
     }
 
+    public String getWarningCode() {
```

Comment:
are these optional values?  In which case what happens when there are no warnings applied on a Task?  What do we return?  There are a few options which probably need to agree and have tests around.  E.g. omit the field altogether, include the field but have it as "null" or empty "" etc.